src/components/Theme/constants.css

```diff
@@ -1,5 +1,5 @@
-@custom-media --large (min-width: 30em);
-@custom-media --small-viewport (max-width: 30em);
+@custom-media --large (min-width: 480px);
+@custom-media --small-viewport (max-width: 479px);
```

Comment:
A better solution might be to keep using `em`s, but to avoid mixing `min-width` and `max-width`

You can't have mutually exclusive rules, but just letting rules override each other is fairly standard in CSS.

But if you've been depending on mutually exclusive rules so far, this might cause issues swapping over to it. So just leaving this comment as an fyi